docs/v2.md

```diff
@@ -0,0 +1,18 @@
+# V2 migration
+
+Changes:
+
+- Context 
+    - container first class citizen 
+    - default mount point in the container is /workspace/.taskctl
+        - if you are referencing an absolute path this 
```

Comment:
Missing README part, but also I think we might need to change this back until we have everything migrated across really

Reply:
@dnitsch